lib/ansible/module_utils/acme.py

```diff
@@ -635,6 +635,8 @@ def get_request(self, uri, parse_json_result=True, headers=None, get_only=False,
             # Perform unauthenticated GET
             resp, info = fetch_url(self.module, uri, method='GET', headers=headers)
 
+            _assert_fetch_url_success(resp, info)
```

Comment:
Can you add another call of this after `send_signed_request`'s `fetch_url` call?

(The `fetch_url` in `get_nonce` does not need one.)